app/src/main/res/values/strings.xml

```diff
@@ -440,9 +440,9 @@ Upload your first media by tapping on the add button.</string>
   <string name="ends_on">Ends on:</string>
   <string name="display_campaigns">Display campaigns</string>
   <string name="display_campaigns_explanation">See the ongoing campaigns</string>
-  <string name="get_content_photo_picker_title">Use GET_CONTENT photo picker</string>
-  <string name="get_content_photo_picker_explanation">Disable if your pictures get uploaded without location</string>
-  <string name="location_loss_warning">Please make sure that this new Android picker does not strip location from your pictures.</string>
+  <string name="open_document_photo_picker_title">Use document based photo picker</string>
+  <string name="open_document_photo_picker_explanation">The new Android photo picker has the potential to strip location information. Enable if you seem to be using it.</string>
```

Comment:
"has the potential to" sounds positive. How about conveying the same in a way that implies it is a bad thing?
Example:
"The new Android photo picker risks losing location information"

Same for the string below.